src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -199,12 +205,32 @@ class PublicationSerializer(serializers.Serializer):
             "GPP-Publicatiebank."
         )
     )
+    gepubliceerd_op = serializers.DateTimeField(
+        help_text=_(
+            "System timestamp reflecting when the publication was published in the "
+            "GPP-Publicatiebank."
+        )
+    )
     laatst_gewijzigd_datum = serializers.DateTimeField(
         help_text=_(
             "System timestamp reflecting when the publication was last modified in the "
             "GPP-Publicatiebank."
         ),
     )
+    datum_begin_geldigheid = serializers.DateTimeField(
+        help_text=_(
+            "The date when the rights and obligations of the attached "
+            "documents come into effect."
+        ),
+        allow_null=True,
+    )
+    datum_einde_geldigheid = serializers.DateTimeField(
+        help_text=_(
+            "The date when the rights and obligations of the attached "
+            "documents stops being in effect."
+        ),
+        allow_null=True,
+    )
```

Comment:
both should be `required=False` to be backwards compatible.

Now existing clients would have to update code to talk to a new version of the GPP-Zoeken API, which means we would have to bump to v3 of the API and that's going a bit too far IMO.